plugin-server/src/main/ingestion-queues/on-event-handler-consumer.ts

```diff
@@ -4,15 +4,21 @@ import { KAFKA_EVENTS_JSON, prefix as KAFKA_PREFIX } from '../../config/kafka-to
 import { Hub } from '../../types'
 import { status } from '../../utils/status'
 import Piscina from '../../worker/piscina'
-import { eachBatchAsyncHandlers } from './batch-processing/each-batch-async-handlers'
+import {
+    eachBatchAppsOnEventHandlers,
+    eachBatchAsyncHandlers,
+    eachBatchWebhooksHandlers,
+} from './batch-processing/each-batch-async-handlers'
 import { KafkaJSIngestionConsumer } from './kafka-queue'
 
-export const startOnEventHandlerConsumer = async ({
+export const startAsyncHandlerConsumer = async ({
```

Comment:
Personal preference: I would keep the start consumer functions separate for each of the `kind`. That way we have the option of making e.g. the webhooks interface much narrower i.e. maybe we could get away with only passing the postgres client in, and thereby avoid needing to load plugins, or clickhouse client for instance.

I would tend to prefer that pattern over pushing the detail into if statements within the function.